addons/hr_expense/tests/test_expenses.py

```diff
@@ -932,3 +932,23 @@ def test_reset_move_to_draft(self):
 
         self.assertEqual(expense.state, 'done', 'Expense state must be done after payment')
         self.assertEqual(expense_sheet.state, 'done', 'Sheet state must be done after payment')
+
+    def test_inverse_total_amount(self):
+        """ Test if the inverse method works correctly """
+
+        expense = self.env['hr.expense'].create({
+            'name': 'Choucroute Saucisse',
+            'employee_id': self.expense_employee.id,
+            'product_id': self.product_a.id,
+            'total_amount': 60,
+            'unit_amount': 0,
+            'tax_ids': [self.tax_purchase_a.id, self.tax_purchase_b.id],
+            'analytic_distribution': {
+                self.analytic_account_1.id: 50,
+                self.analytic_account_2.id: 50,
+            },
+        })
+
+        expense.total_amount = 90
+
+        self.assertEqual(expense.unit_amount, 90, 'Unit amount should be the same as total amount was written to')
```

Comment:
The CI style is failing because of a missing EOF
```suggestion
        self.assertEqual(expense.unit_amount, 90, 'Unit amount should be the same as total amount was written to')

```

Reply:
Added EOF newline